tests/devices/test_default_clifford.py

```diff
@@ -284,7 +285,7 @@ def circuit_fn():
 
     gotten_probs, target_probs = qnode_clfrd(), qnode_qubit()
 
-    assert qml.math.allclose(gotten_probs, target_probs, atol=1e-2 if shots else 1e-8)
+    assert qml.math.allclose(gotten_probs, target_probs, atol=5e-1 if shots else 1e-8)
```

Comment:
Why is the tolerance changed to `0.5`? This seems quite large.

Reply:
I guess since, otherwise, it fails too frequently due to random fluctuations

Reply:
I had a similar issue in my previous PR

Reply:
I guess this is because, otherwise, it fails too frequently due to random fluctuations. I had a similar issue in my previous PR

Reply:
The tests were passing consistently before with `1e-2` so what changed?

Reply:
For some bizarre reason, I have the feeling that with opmath enabled, such errors occur more often. I don't know if this is the case but for example, in my PR a similar test was failing quite often before I incremented the tolerance

Reply:
yeah honestly was also a bit puzzled by this but the result seem to match/be true still. It could be related to something about how things are sampled, but I dont know too much about that part of the codebase.

Reply:
For now I'm ok with setting the tolerance higher, but probably open a story to document this phenomenon so that it can be investigated in the future? I'm a bit concerned that we could be covering up an actual issue with such a large tolerance. Or have you tested the failure rate of the test cases with a smaller tolerance like the standard `0.05` that is used by most other tests involving random processes?

Reply:
Yes, good point I opened a story here https://app.shortcut.com/xanaduai/story/59140

Reply:
Actually I did some testing locally and it seems that a tolerance of `0.05` would allow the test case to consistently pass. I would be ok with that.

Reply:
Oh I think that is actually what I wanted but went one order of magnitude too far. I updated the test, though I think this could still be worth investigating if there are any differences now in sampling with new opmath